server/auth/token.js

```diff
@@ -2,12 +2,22 @@ const { promisify } = require('util')
 const crypto = require('crypto')
 
 const randomBytes = promisify(crypto.randomBytes)
-const tokenKey = Buffer.from(process.env.APP_TOKEN_KEY, 'base64')
+const tokenKey = Buffer.from(process.env.RCTF_TOKEN_KEY, 'base64')
```

Comment:
I think we should use `config.tokenKey` instead of hardcoding environmental variables